### PR TITLE
[Rule Tuning] LSASS Process Access via Windows API

### PR DESCRIPTION
**Pull request by joe-desimone**
Tuning for `agentbeat` false positives observed with 8.14.X for rule `LSASS Process Access via Windows API`

### CODE REVIEW

File: rules/windows/credential_access_lsass_openprocess_api.toml [hunk 1]
```diff
@@ -137,6 +137,7 @@ api where host.os.type == "windows" and
         "?:\\Program Files\\Cisco\\AMP\\*\\sfc.exe",
         "?:\\Program Files\\Common Files\\McAfee\\AVSolution\\mcshield.exe",
         "?:\\Program Files\\EA\\AC\\EAAntiCheat.GameService.exe",
+        "?:\\Program Files\\Elastic\\Agent\\data\\elastic-agent-*\\components\\agentbeat.exe",
```
Comment (terrancedejesus) [inline rules/windows/credential_access_lsass_openprocess_api.toml:140]: @joe-desimone can you update the updated date when you get a chance before merging.

**Review by terrancedejesus — APPROVED** [review]

File: rules/windows/credential_access_lsass_openprocess_api.toml (hunk 1 above)
Comment (Samirbous) [reply rules/windows/credential_access_lsass_openprocess_api.toml:140]: done

**Review by Samirbous — APPROVED** [review]

**Review by Mikaayenson — COMMENTED** [review]
@joe-desimone just need to modify the `updated_date` and it should be g2g.

**Comment by Samirbous** [comment]
added few more noisy FP patterns including one reported in community slack channel.

**Pull request merged**